### PR TITLE
chore(appup): Add missing module

### PR DESCRIPTION
**Pull request by ieQu1**

### CODE REVIEW

**Review by qzhuyan — APPROVED** [review]

**Comment by qzhuyan** [comment]
do you need to bump vsn?

**Comment by ieQu1** [comment]
> do you need to bump vsn?

We use `{version, git}` in gen_rpc, so it's updated automatically.

File: src/gen_rpc.appup.src [hunk 1]
```diff
@@ -3,6 +3,7 @@
 {VSN,
   [{<<"2.8.1">>,
```
Comment (zmstone) [inline src/gen_rpc.appup.src:4]: if you are about to tag 2.8.3, then 2.8.2 should be added here.

Comment (zmstone) [reply src/gen_rpc.appup.src:4]: nvm.

**Pull request merged**